### PR TITLE
plugin: renew 6.0 summoner job detail

### PR DESCRIPTION
**Pull request by MaikoTan**
Still no idea what is 0x04-0x05 is.
But seems like that we need is already here?

### CODE REVIEW

**Comment by MaikoTan** [comment]
Summoner are basically renewed fully, how can I get them still work on CN/KR but also keep TypeScript happy with new parameters in job details? @quisquous

**Comment by quisquous** [comment]
> Summoner are basically renewed fully, how can I get them still work on CN/KR but also keep TypeScript happy with new parameters in job details? @quisquous

Maybe a discriminated union, where you have both sets of fields { oldFields } | { newFields } but only one set of them is true?

**Comment by MaikoTan** [comment]
Added a blank SMNComponent for now, maybe Echoring or me would do this later.

File: ui/jobs/components/smn.ts [hunk 1]
```diff
@@ -74,13 +79,13 @@ export class SMNComponent extends BaseComponent {
   }
 
   override onYouGainEffect(id: string, matches: PartialFieldMatches<'GainsEffect'>): void {
-    if (id === EffectId.FurtherRuin) {
+    if (id === '4BC') { // Further Ruin changed id in 6.0, EffectId.FurtherRuin is the 6.0 version.
```
Comment (quisquous) [inline ui/jobs/components/smn.ts:82]: Could you add `FurtherRuin5x: 4BC` into ui/jobs/constants?

Comment (MaikoTan) [reply ui/jobs/components/smn.ts]: Done!

**Pull request merged**